src/components/modes/teacher/BackgroundToggle.js

```diff
@@ -61,7 +96,7 @@ const BackgroundToggle = () => {
         control={
           <Switch
             color="primary"
-            checked={backgroundImage?.visible ?? false}
+            checked={backgroundToggleSetting?.data.toggle}
```

Comment:
`backgroundToggleSetting?.data?.toggle` otherwise it will throw for `undefined.toggle`